cax/config.py

```diff
@@ -5,17 +5,22 @@
 import logging
 import os
 import socket
+import subprocess
 
 import pymongo
 
 # global variable to store the specified .json config file
 CAX_CONFIGURE = ''
 DATABASE_LOG = True
 
-PAX_DEPLOY_DIRS = {
-    'midway-login1' : '/project/lgrandi/deployHQ/pax',
-    'tegner-login-1': '/afs/pdc.kth.se/projects/xenon/software/pax'
-}
+
+
+def pax_deploy_directories(host, pax_version):
+    PAX_DEPLOY_DIRS = {
+      'midway-login1' : '/project/lgrandi/deployHQ/pax',
+      'tegner-login-1': '/afs/pdc.kth.se/projects/xenon/software/pax/pax_'+pax_version
```

Comment:
Why do you need to specify pax version here? You should have one anaconda setup that has different environments for different pax versions.  That's why this wasn't a function.